src/index.ts

```diff
@@ -1,4 +1,7 @@
-export {TitleEscrowFactory} from "./contracts/TitleEscrowFactory";
-export {TradeTrustErc721Factory} from "./contracts/TradeTrustErc721Factory";
-export {TitleEscrowCreatorFactory} from "./contracts/TitleEscrowCreatorFactory";
-export {getTitleEscrowCreatorAddress} from "./config";
+/* eslint-disable camelcase */
+export {
+  TitleEscrow__factory as TitleEscrowFactory,
+  TitleEscrowCreator__factory as TitleEscrowCreatorFactory,
+  TradeTrustERC721__factory as TradeTrustErc721Factory,
```

Comment:
this i sa breakin change, the name changed :)

Reply:
ERC vs Erc

Reply:
Yes, the original master branch is exported as `Erc`: https://github.com/Open-Attestation/token-registry/blob/c8a39d2b57b4e6a29bce54a59db436ed424509dc/src/index.ts#L2

Just not too sure why the generated filenames automatically capitalised `ERC`.

Reply:
yes so it's a breking change right ?
- before Erc
- now ERC

I don't mind the change, but if it's a breaking change we must release it that way :)

Reply:
wanna smack the typechain people